src/embind/embind.js

```diff
@@ -278,11 +278,8 @@ var LibraryEmbind = {
     });
   },
 
-  _embind_register_bool__deps: [
-    '$getShiftFromSize', '$readLatin1String', '$registerType'],
-  _embind_register_bool: function(rawType, name, size, trueValue, falseValue) {
-    var shift = getShiftFromSize(size);
-
+  _embind_register_bool__deps: ['$readLatin1String', '$registerType'],
+  _embind_register_bool: function(rawType, name, trueValue, falseValue) {
```

Comment:
It's unused but we should also drop the size parameter in [embind_ts.js](https://github.com/emscripten-core/emscripten/blob/main/src/embind/embind_ts.js#L270)

Reply:
Done